controllers/drplacementcontrol.go

```diff
@@ -264,6 +269,7 @@ func (d *DRPCInstance) startDeploying(homeCluster, homeClusterNamespace string)
 
 	err = d.updateUserPlacementRule(homeCluster, homeClusterNamespace)
 	if err != nil {
+		err := fmt.Errorf("failed to update UserPlacementRule %s", d.userPlacementRule.Name)
```

Comment:
Why do we need this? `updateUserPlacementRuleStatus` returns the said error string and also the error from the API server that was received.

Reply:
We also seem to have added an event in `updateUserPlacementRuleStatus` which would hence be a duplicate.